src/Chainweb/Chainweb.hs

```diff
@@ -674,7 +690,7 @@ runChainweb cw = do
 
     -- collect server resources
     proj :: forall a . (ChainResources logger -> a) -> [(ChainId, a)]
-    proj f = map (second f) chains
+    proj f = chains & mapped . _2 %~ f
```

Comment:
This is a bit gratuitously unnecessary. The old form was actually clearer to me.

Reply:
seconded